src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Invoice/InvoicePaymentsResource.cs

```diff
@@ -0,0 +1,35 @@
+﻿using SwedbankPay.Sdk.Extensions;
+using SwedbankPay.Sdk.Payments.InvoicePayments;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments
+{
+    public class InvoicePaymentsResource : ResourceBase, IInvoiceResource
+    {
+        public InvoicePaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public async Task<IInvoicePayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)